config-templates/redhat/odf-local/4.7/README.md

```diff
@@ -16,6 +16,40 @@ In order to deploy ODF, the following prerequisites are required.
 - [Add your Satellite to a cluster group](cloud.ibm.com/docs/satellite?topic=satellite-cluster-config#setup-clusters-satconfig-groups).
 - [Optional] You must provision an instance of IBM Cloud Object Storage and provide your COS HMAC credentials, the regional public endpoint, and the IBM COS location when you create your storage configuration if you want to use ibm cos as the default backingstore. If not provided, pv-pool will be used.
 
+### Creating the satellite link for the private container endpoint and the storage secret
+
+1) Create the satellite link for the container api private endpoint
+From the satellite location UI, create a link with
+a) Destination FQDN or IP : private.<us-east>.containers.cloud.ibm.com    [Replace us-east with the region of your satellite location]
+b) Destination port : 443
+
+Sample satellite link :  https://s846feeb2f2e56cfc88a1-6b64a6ccc9c596bf59a86625d8fa2202-c000.us-east.satellite.appdomain.cloud:32232
+
+Please provide this link for the parameter : `container-private-endpoint`
+
+2) Create the storage secret on your satellite cluster
+Create the storage secret or update the secret if already present
+
+```       
+apiVersion: v1
+kind: Secret
+metadata:
+      name: storage-secret-store
+      namespace: kube-system
+type: Opaque
+stringData:
+      slclient.toml: |-
+        [Bluemix]
+          iam_url = "https://iam.bluemix.net"
+          iam_client_id = "bx"
+          iam_client_secret = "bx"
+          iam_api_key = "<Please replace this with your IAM API Key>"
+          containers_api_route_private = "<Please replace this with the satellite link created in step 1>"
+
+        [VPC]
+          provider_type = "g2"
```

Comment:
please check with Sagar if this is needed or we can remove as VPC specific things are not needed I guess.

Reply:
This is still required for now